index.d.ts

```diff
@@ -130,6 +156,8 @@ export class Notification {
    * This Object is JSON encoded and sent as the notification payload. When properties have been set on notification.aps (either directly or with convenience setters) these are added to the payload just before it is sent. If payload already contains an aps property it is replaced.
    */
   public payload: any;
+  public aps: Aps;
```

Comment:
@florianreinhart regarding `aps`, it does mention this in [README.md](https://github.com/node-apn/node-apn/blob/master/doc/notification.markdown#notificationpayload)

So it should be exposed, or am I mistaken here?